Firebase/Core/FIRBundleUtil.m

```diff
@@ -52,10 +52,10 @@ + (BOOL)hasBundleIdentifierPrefix:(NSString *)bundleIdentifier inBundles:(NSArra
     // This allows app extensions that have the app's bundle as their prefix to pass this test.
     NSString *applicationBundleIdentifier =
         [GULAppEnvironmentUtil isAppExtension]
-            ? [self bundleIdentifierByRemovingLastPartFrom:bundleIdentifier]
+            ? [self bundleIdentifierByRemovingLastPartFrom:bundle.bundleIdentifier]
             : bundleIdentifier;
 
-    if ([applicationBundleIdentifier isEqualToString:bundle.bundleIdentifier]) {
+    if ([applicationBundleIdentifier isEqualToString:bundleIdentifier]) {
```

Comment:
Isn't this equivalent to `if (![GULAppEnvironmentUtil isAppExtension])`?

Reply:
I guess they still want to double check the bundle ID of app extension has the same prefix as the host/containing app.

Reply:
The unit test expect the app extension target to pass in a googleService-Info.plist that is created specifically for extension, not hosting app.

@renkelvin which scenario is it when testing auth in app extension? Do you use the same GoogleService-Info.plist or a new one for extension?

Reply:
@chliangGoogle There is no extension target for Auth yet.

Reply:
talked to @renkelvin offline, seems like this doesn't impact Auth. It comes to how Core like the start up flow be for app extension users. @ryanwilson Based on our discussion, I think the fix is what you want for start up flow.